src/domain/entities/Note.ts

```diff
@@ -14,6 +19,12 @@ export type NoteContent = OutputData;
  * Note entity
  */
 export interface Note {
+
+  /**
+   * Note public id
+   */
+  publicId: NotePublicId;
```

Comment:
note should now contain `publicId`. For the frontend-side there is only "id" which is public `id`.

Only Backend knows that note have both "public id" and "internal id"

Reply:
you mean, there should only be publicId? no other ids?

Reply:
yes, but public id should be named just "id". 

The idea is to completely hide internal "id" from API users. So they could not loop over it and check how many articles we have. So "public" prefix makes sense only for backend. For frontend, there is only "note.id" (which is public id)